jailctl/blogin

```diff
@@ -68,7 +68,7 @@ login_internal()
 	fi
 
 	export TERM=xterm
-	/usr/local/bin/tmux attach-session -t cbsdb-${jname} 2>/dev/null || /usr/local/bin/tmux attach-session -t cbsd-${jname}
+	"$(which tmux)" -Lcbsd-"${jname}" attach-session
```

Comment:
Why are you using `which`? I mean, if it's in PATH, just plain `tmux` will be fine. If it isn't, `which` doesn't help. Same for the changes below.

Reply:
bad influence `bhyve.subr`, deleted